src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -795,6 +814,32 @@ LeaseStatusTracker::LeaseStatusTracker(
   }
 }
 
+std::shared_ptr<LeaseStatusTracker> LeaseStatusTracker::CreatePrepared(
+    std::shared_ptr<GcsPlacementGroup> placement_group,
+    const std::vector<std::shared_ptr<const BundleSpecification>> &prepared_bundles) {
```

Comment:
No need to pass in `prepared_bundles` since you can get it from `placement_group`?